From: Michael Tokarev <mjt@tls.msk.ru>
Subject: add -fno-ipa-sra to seabios-hppa compiler flags
Date: Wed, 22 Jul 2020 22:15:46 +0300

This allows seabios-hppa to build with gcc-10. Or else the
compiler generates eg memset.isra.0 symbols instead of memset,
and the final link step fails due to missing memset.

index c0d5d958..1b7757e8 100644
--- a/roms/seabios-hppa/Makefile.parisc
+++ b/roms/seabios-hppa/Makefile.parisc
@@ -76 +76 @@ COMMONCFLAGS := -I$(OUT) -Isrc -Ivgasrc -Os -MD -g \
-    -fno-builtin-printf
+    -fno-builtin-printf -fno-ipa-sra
